commit ae94046d8654b9f65df871502de0f08a36aa5d38
Author: Julien Cristau <jcristau@debian.org>
Date:   Tue Aug 19 01:04:40 2008 +0200

    xaw8 is gone, use xaw7

diff --git a/configure.ac b/configure.ac
index 2831440..318a2c6 100644
--- a/configure.ac
+++ b/configure.ac
@@ -34,7 +34,7 @@ AC_PROG_INSTALL
 AC_CHECK_FUNCS([nanosleep poll select])
 
 # Checks for pkg-config packages
-XAW_CHECK_XPRINT_SUPPORT(XMAG)
+PKG_CHECK_MODULES(XMAG, xaw7)
 AC_SUBST(XMAG_CFLAGS)
 AC_SUBST(XMAG_LIBS)
 
